README.FreeBSD

```diff
@@ -0,0 +1,56 @@
+
+The FreeBSD build will build most of the tools in Ceph.
+Note that the RBD dependant items will not work since FreeBSD does not have RBS
+
+Build Prerequisites
+===================
+
+Compiling and building Ceph is tested on 11-CURRENT 
+It uses the CLANG toolset taht is available, which needs to be atleast 3.7
+Clang 3.4 (on 10.2-STABLE) does not have all required capabilites to compile everything
+
+The following setup will get things running for FreeBSD:
+
+ - Install bash and link it in /bin
+	# requires root privileges
+	sudo pkg install bash
+	sudo ln -s /usr/local/bin/bash /bin/bash
+
+Building Ceph
+=============
+ - Go and start building
+	./do_freebsd.sh
+
+Parts not (yet) included:
+=========================
+
+ - RBD
+   Rados Block Devices is implemented in the Linux kernel
+   It seems that there used to be a userspace implementation first.
+   And perhaps ggated could be used as a template since it does some of the
+   same, other than just between 2 disks. And it has a userspace counterpart.
```

Comment:
FYI, I started implementation based on geom_gate but it got stack, partially due to other activity, but also due to continuous frustration when build breaks after next master rebase.